documentation/source/library_reference.rst

```diff
@@ -282,3 +281,24 @@ Signal Tracer for WaveDrom
     :members:
     :member-order: bysource
     :synopsis: A signal tracer for WaveDrom.
+
+
+Developer-focused
+=================
```

Comment:
For reviewing: https://external-builds.readthedocs.io/html/cocotb/1179/library_reference.html#developer-focused

Reply:
I don't immediately see why ``fork()`` is still not linked... :/

Reply:
It is linked?

Reply:
Indeed it is linked, maybe some caching glitch.